resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/hadooputils/HadoopKerberosLogin.scala

```diff
@@ -48,10 +48,8 @@ private[spark] object HadoopKerberosLogin {
        submissionSparkConf,
        hadoopConf)
      require(tokenData.nonEmpty, "Did not obtain any delegation tokens")
-     val currentTime = tokenManager.getCurrentTime
-     val initialTokenDataKeyName = s"$KERBEROS_SECRET_KEY_PREFIX-$currentTime"
-     val newSecretName =
-       s"$kubernetesResourceNamePrefix-$KERBEROS_DELEGEGATION_TOKEN_SECRET_NAME.$currentTime"
+     val initialTokenDataKeyName = KERBEROS_SECRET_KEY_PREFIX
```

Comment:
Nit: you don't need this variable and you can rename `KERBEROS_SECRET_KEY_PREFIX` to `KERBEROS_SECRET_KEY`.